hugegraph-llm/src/hugegraph_llm/config/config.py

```diff
@@ -106,13 +106,13 @@ def ensure_yaml_file_exists(self):
 
 
     def save_to_yaml(self):
-        indented_schema = "\n".join([f"  {line}" for line in self.rag_schema.splitlines()])
-        indented_example_prompt = "\n".join([f"    {line}" for line in self.schema_example_prompt.splitlines()])
-        indented_question = "\n".join([f"    {line}" for line in self.question.splitlines()])
+        indented_schema = "\n".join([f"  {line}" for line in self.graph_schema.splitlines()])
+        indented_example_prompt = "\n".join([f"    {line}" for line in self.extract_graph_prompt.splitlines()])
+        indented_question = "\n".join([f"    {line}" for line in self.default_question.splitlines()])
         indented_custom_related_information = (
-            "\n".join([f"    {line}" for line in self.custom_related_information.splitlines()])
+            "\n".join([f"    {line}" for line in self.custom_rerank_info.splitlines()])
         )
-        indented_default_answer_template = "\n".join([f"    {line}" for line in self.default_answer_template.splitlines()])
+        indented_default_answer_template = "\n".join([f"    {line}" for line in self.answer_prompt.splitlines()])
 
         # This can be extended to add storage fields according to the data needs to be stored
         yaml_content = f"""rag_schema: |
```

Comment:
The prompt name in this section needs to be changed, so that the yaml file stores the correct information